conformance/src/main/scala/za/co/absa/enceladus/conformance/interpreter/DynamicInterpreter.scala

```diff
@@ -45,32 +44,16 @@ object DynamicInterpreter {
     * @return The conformed dataframe
     *
     */
-  def interpret(conformance: ConfDataset,
-                inputDf: Dataset[Row],
-                jobShortName: String = "Conformance",
+  def interpret(conformance: ConfDataset, inputDf: Dataset[Row], jobShortName: String = "Conformance",
                 experimentalMappingRule: Boolean)
                (implicit spark: SparkSession, dao: EnceladusDAO, progArgs: CmdConfig, enableCF: Boolean): DataFrame = {
```

Comment:
`enableCF` is an implicit boolean parameter. This might be easily cause errors if we have another method that having an implicit boolean which has a different meaning. Consider making this parameter explicit.